src/main/java/de/hpi/bpt/argos/api/eventQuery/EventQueryEndpointImpl.java

```diff
@@ -1,44 +1,232 @@
-package de.hpi.bpt.argos.api.eventQuery;
-
-import de.hpi.bpt.argos.api.eventType.EventTypeEndpoint;
-import de.hpi.bpt.argos.api.response.ResponseFactory;
-import de.hpi.bpt.argos.common.RestEndpointImpl;
-import de.hpi.bpt.argos.persistence.database.PersistenceEntityManager;
-import spark.Request;
-import spark.Response;
-import spark.Service;
-
-/**
- * {@inheritDoc}
- * This is the implementation.
- */
-public class EventQueryEndpointImpl extends RestEndpointImpl implements EventQueryEndpoint {
-
-	/**
-	 * {@inheritDoc}
-	 */
-	@Override
-	public void setup(ResponseFactory responseFactory, PersistenceEntityManager entityManager, Service sparkService) {
-		super.setup(responseFactory, entityManager, sparkService);
-		sparkService.post(EventQueryEndpoint.getUpdateEventQueryBaseUri(), this::updateEventQuery);
-	}
-
-	/**
-	 * {@inheritDoc}
-	 */
-	@Override
-	public String updateEventQuery(Request request, Response response) {
-		logInfoForReceivedRequest(request);
-
-		long eventTypeId = inputValidation.validateLong(
-				request.params(EventTypeEndpoint.getEventTypeIdParameter(false)),
-				(Long input) -> input > 0);
-
-		responseFactory.updateEventQuery(eventTypeId, request.body());
-
-		logInfoForSendingResponse(request);
-		return responseFactory.finishRequest();
-	}
-
-
-}
+package de.hpi.bpt.argos.api.eventQuery;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonSyntaxException;
+import de.hpi.bpt.argos.api.eventType.EventTypeEndpoint;
+import de.hpi.bpt.argos.api.eventType.EventTypeEndpointImpl;
+import de.hpi.bpt.argos.common.EventPlatformFeedback;
+import de.hpi.bpt.argos.common.EventProcessingPlatformUpdaterImpl;
+import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
+import de.hpi.bpt.argos.storage.dataModel.event.query.EventQuery;
+import de.hpi.bpt.argos.storage.dataModel.event.query.EventQueryImpl;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventType;
+import de.hpi.bpt.argos.util.HttpStatusCodes;
+import de.hpi.bpt.argos.util.LoggerUtilImpl;
+import de.hpi.bpt.argos.util.RestEndpointUtil;
+import de.hpi.bpt.argos.util.RestEndpointUtilImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.HaltException;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+import java.util.List;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)